### PR TITLE
modulesync 0.20.1

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: spec/acceptance/nodesets/docker/ubuntu-12.04.yml [hunk 1]
```diff
@@ -12,8 +12,6 @@ HOSTS:
     docker_image_commands:
       - 'apt-get install -y net-tools wget'
       - 'locale-gen en_US.UTF-8'
-      - 'ln -s /opt/puppetlabs/bin/facter /usr/bin/facter'
```
Comment (vinzent) [inline spec/acceptance/nodesets/docker/ubuntu-12.04.yml]: @petems you added these 2 symlinks in the modulesync managed file with https://github.com/voxpupuli/puppet-collectd/commit/8dba7fc508696e72d8e513c7b62520960eff5a57 . is it really required?

Comment (petems) [inline spec/acceptance/nodesets/docker/ubuntu-12.04.yml]: @vinzent I think depending on the version of beaker used, the path doesn't get exported and it's needed. Let me check.

Comment (bastelfreak) [inline spec/acceptance/nodesets/docker/ubuntu-12.04.yml]: ping @petems

**Comment by bastelfreak** [comment]
I'm going to merge this as is. We can still change it in the future if we notice that the symlinks are required.

**Pull request merged**